apps/demo/app/(root)/(tab).js

```diff
@@ -1,8 +1,8 @@
 import { Tabs } from "expo-router";
 export default function Layout1() {
   return (
-    <Tabs>
-      <Tabs.Screen name="settings" />
+    <Tabs initialRouteName={"index"}>
+      <Tabs.Screen name="settings" options={{ title: "Settings", headerShown: false }} />
     </Tabs>
```

Comment:
```suggestion
    <Tabs>
      <Tabs.Screen name="index" />
      <Tabs.Screen name="settings" options={{ title: "Settings", headerShown: false }} />
    </Tabs>
```

Ordering is done by using the children in the layout.

Reply:
its not about order, its about true parent in linking, when app start with deep link modal
![image](https://user-images.githubusercontent.com/839633/193696119-53dd996f-f61a-49b0-952e-d43dcf860149.png)
![image](https://user-images.githubusercontent.com/839633/193696133-cb5043f4-910c-411a-9013-320454c83539.png)

Reply:
🙋‍♂️ how'd you make this half-screen modal with detents? is it using native stack?

Reply:
@nandorojo no ) its just screenshot to show who the parent